scripts/mk/go-rules.mk

```diff
@@ -60,8 +60,18 @@ run: $(BIN)/service .compose-wait-db ## Run the api & kafka consumer locally
 # See: https://go.dev/doc/modules/managing-dependencies#synchronizing
 .PHONY: tidy
 tidy:  ## Synchronize your code's dependencies
-	go mod tidy
-	cd tools && go mod tidy
+	go mod tidy -go=$(GO_VERSION)
+	cd tools && go mod tidy -go=$(GO_VERSION)
+
+.PHONY: container-go-tidy
+container-tidy:
+	$(CONTAINER_ENGINE) run \
+	  --rm \
+	  --userns=keep-id:uid=1001,gid=1001 \
+	  -v $(PROJECT_DIR):/opt/app-root/src/idmsvc-backend:Z \
+	  -w /opt/app-root/src/idmsvc-backend \
+	  $(GO_TOOLSET_CONTAINER) \
+	  make tidy
```

Comment:
I'd rather to avoid leverage of container on this file, and I think the change in `tidy` rule would be enough.